package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "escape-html",
-  "description": "Escape HTML entities",
+  "description": "Escape HTML special characters (syntax)",
```

Comment:
> This is no corresponding change in `component.json` to match this change.

Ok, if we come up with new wording that you like I'll open a new PR that updates that as well.

> What does putting "(syntax)" add to the description? I'm also not sure many people understand what a "special character" is, so it should be a very simplistic description (but can easily be expanded in the readme).

Including "syntax" in the description is just an extra attempt to explain what this function changes in case they don't understand "special character" (the syntax characters are "special"). "Special character" isn't a formal term in HTML, so perhaps "escape string for use in HTML" is a better description, especially if the premise is that people know that something needs to be escaped, but have no idea what ("escape HTML [syntax|markup] characters" would probably be the most accurate). But it's certainly better than the previous description, that didn't make sense: "escape HTML entities".